dev/devicelab/lib/framework/metrics_center.dart

```diff
@@ -70,7 +70,12 @@ List<MetricPoint> parse(Map<String, dynamic> resultsJson) {
   return metricPoints;
 }
 
-/// Upload test metrics to metrics center.
+/// Upload JSON results to metrics center.
+///
+/// Flutter infrastructure's workflow is:
+/// 1. Run DeviceLab test, writing results to a known path
+/// 2. Request service account token from luci auth (valid for at least 3 minutes)
+/// 3. Upload results from (1) to metrics center.
```

Comment:
Can we explicitly call metrics center skia infra? There are many metrics centers, and its hard to know which one this refers to.